### ISSUE TITLE
Resets for UserAgent and RateLimits

### ISSUE THREAD

**Issue by C9Glax** [C9Glax/tranga-website#59]
I added a Reset for the UserAgent and RequestLimits for completness
`POST Settings/customRequestLimit/Reset`
`POST Settings/userAgent/Reset`

**Comment by db-2001** [comment]
Does the request limit reset do all of them?

**Comment by C9Glax** [comment]
Yes, would you want a Seperate button for each?
Would not be too difficult either i think.
However I put this in a seperate issue to the current PR to get that over with for now 👍🏼

**Comment by db-2001** [comment]
Yeah we can address this in the next PR, I don't want separate for each, maybe just separate for Mangadex and separate for the other defaults

**Comment by C9Glax** [comment]
#159 Server-V2
